CentralComputing/Pod_State.cpp

```diff
@@ -209,13 +213,54 @@ void Pod_State::steady_launch_ready(std::shared_ptr<NetworkManager::Network_Comm
 }
 
 void Pod_State::steady_flight_accelerate(std::shared_ptr<NetworkManager::Network_Command> command) {
+	// Access Pos, Vel, and Accel from Motion Model
+	double MAX_DECCEL = 19.6;
+	double IDEAL_DECCEL = 9.8;
+	double LENGTH_OF_TRACK = 1000;
+	double BUFFER_LENGTH = 20;
+	double MAX_VELOCITY = 550;
+	std::shared_ptr<StateSpace> state = SourceManager::MM.Get();
+	double pos = state->x[0];
+	double vel = state->x[1];
+	double acc = state->x[2];
+	
+	if (-.5*vel*vel/IDEAL_DECCEL+vel*vel/IDEAL_DECCEL >= LENGTH_OF_TRACK - BUFFER_LENGTH - pos) {
```

Comment:
Refactor this into "ShouldBrake(double pos, double vel)"

Reply:
We should probably set up some unit tests to test this function for certain known values